action.yml

```diff
@@ -270,6 +270,30 @@ inputs:
     required: false
     default: "true"
 
+  clang_format:
+    description: Enable or disable ClangFormat checks
+    required: false
+    default: "false"
+  clang_format_args:
+    description: Additional arguments to pass to the linter
+    required: false
+    default: ""
+  clang_format_dir:
+    description: Directory where the ClangFormat command should be run
+    required: false
+  clang_format_extensions:
+    description: Extensions of files to check with ClangFormat
+    required: false
+    default: "c,cc,cpp,h,hpp,m,mm"
```

Comment:
Actually ClangFormat supports more various filetypes like C#, JS, JSON, Verilog and so on. But I think the most popular use case of ClangFormat is formatting C language family.